app/app/mailers/weekly_report_mailer.rb

```diff
@@ -0,0 +1,61 @@
+require "csv"
+class WeeklyReportMailer < ApplicationMailer
+  helper_method :week_start_date_formatted, :prev_week_start_date_formatted
+
+  # Send email with a CSV file that reports on completed flows in past week
+  def report_email
+    email_address = site_config["nyc"].transmission_method_configuration["email"]
+    site_id = "nyc"
+    now = Time.now
+    previous_week_start_date = now.beginning_of_week - 7.days
```

Comment:
This might work better
```suggestion
previous_week_start_date = now.beginning_of_week.prev_week
```